stratagems/lang/polish/setup.tra

```diff
@@ -1,4 +1,5 @@
 @4 = ~Ulepszenia rozgrywki~
+@5 = ~Instaluj w trybie batchowym (instalator pyta o wszystkie skladniki przed rozpoczeciem instalacji). NIE NALEZY UZYWAC Z PROJECT INFINITY.~
```

Comment:
Just a comment, "batch file" is usually translated as "plik wsadowy". It may sound strange if you're unfamiliar with the term, but if you look it up (e.g. in Google), you'll see that this is what IT-related sources actually use, rather than something like "batchowy".

Considering the above, I'd suggest to use "w trybie wsadowym" in the above.

Reply:
Yeah, I know the term "plik wsadowy", but I left "tryb batchowy" as it is as meaningless as "tryb wsadowy", because here it should be something like "w trybie serii". Do you have any better idea?

Reply:
Not really, but the same can be said about the English version, can't it 😉? "Batch mode" means nothing for non-technical people, but you have the explanation right there in the brackets, so I don't think there's actually any risk of misunderstanding here.

For the record, Microsoft (e.g. https://docs.microsoft.com/pl-pl/dynamicsax-2012/appuser-itpro/batch-processing-overview) and IBM (e.g. https://www.ibm.com/docs/pl/spss-modeler/18.3.0?topic=execution-introduction-batch-mode) seem to always translate "batch" as "wsadowy".